src/main/kotlin/org/dashevo/dapiclient/DapiClient.kt

```diff
@@ -264,27 +202,78 @@ class DapiClient(val masternodeService: MasternodeService, val diffMasternodeEac
         }
 
         return response.block.toByteArray()
-    }
+    }*/
 
-    fun sendTransaction(txBytes: ByteString, allowHighFees: Boolean = false, bypassLimits: Boolean = false): String {
-        logger.info("sendTransaction(${txBytes.toByteArray().toHexString()}, allowHighFees=$allowHighFees, bypassLimits=$bypassLimits): jRPC")
-        val request = CoreOuterClass.SendTransactionRequest.newBuilder()
-                .setTransaction(txBytes)
-                .setAllowHighFees(allowHighFees)
-                .setBypassLimits(bypassLimits)
-                .build()
-        val service = getCoreService()
+    /*private fun getBlock2(request: CoreOuterClass.GetBlockRequest?): ByteArray? {
```

Comment:
Do we need this commented code?

Reply:
We don't

Reply:
https://github.com/dashevo/dapi-client-android/pull/7/commits/5e5e9bf568fb1d468647a5077f4a8385f45f7acf